packages/amplify-cli/src/index.ts

```diff
@@ -33,9 +33,36 @@ import { logInput } from './conditional-local-logging-init';
 
 EventEmitter.defaultMaxListeners = 1000;
 
+// Change stacktrace limit to max value to capture more details if needed
+Error.stackTraceLimit = Number.MAX_SAFE_INTEGER;
+
+let errorHandler = (e: Error) => {};
+
+process.on('uncaughtException', function (error) {
+  // Invoke the configured error handler if it is already configured
+  if (errorHandler) {
+    errorHandler(error);
+  } else {
```

Comment:
I don't think the else case is ever going to execute? errorHandler is defined above and reassigned later to boundErrorHandler

Reply:
This is for the ultimate fallback when we would hit any error before that is being assigned like error during context initialization, or ff or something else, it is there to be future proof make sure we are not failing on the failing path.